docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,147 @@
+<!--
+order: 3
+-->
+
+# Quick Start
+
+**Bootstrap a  `cosmoshub-4` mainnet node**
+
+### Prerequisites
+> **Note**: Make sure the [Gaia CLI is installed](./installation.md).
+
+### Sync Options
+Node operators can choose to sync via State Sync of Quicksync. State Sync works by replaying larger chunks of application state directly rather than replaying individual blocks or consensus rounds. Quicksync is a service provided courtesy of Chainlayer, and offers historical state of the chain available for download every 24 hours. For more information on both see the Sync Options section of the full [Joining Mainnet Tutorial](../hub-tutorials/joining-mainnet.md)
+
+<!-- #sync options -->
+::::::: tabs :options="{ useUrlFragment: false }"
+
+:::::: tab "State Sync"
+
+To enable state sync, visit an [explorer](https://www.mintscan.io/cosmos/blocks) to get a recent block height and corresponding hash. A node operator can choose any height/hash in the current bonding period, but as the recommended snapshot period is 1000 blocks, it is advised to choose something close to current height - 1000. Set these parameters in the code snippet below `<BLOCK_HEIGHT>` and `<BLOCK_HASH>`
```

Comment:
Do we need to get people to pick blocks rounded down to `1000`? As in, what if there's no snapshots for blocks ending in xxxxx249 for example? Actually not sure how this works tho.

Reply:
No the trusted block height doesn't need to be the same as the height at which the snapshot was taken. I believe state sync works as long as your node can find a snapshot at a height > trusted height.

Reply:
Cool, thanks for clarifying, appreciate it!